drivers/bluetooth/hci/ipm_stm32wb.c

```diff
@@ -405,17 +405,6 @@ static void start_ble_rf(void)
 
 	/* Select wakeup source of BLE RF */
 	LL_RCC_SetRFWKPClockSource(LL_RCC_RFWKP_CLKSOURCE_LSE);
-
-	/* Switch OFF LSI */
-	LL_RCC_LSI2_Disable();
-#else
-	LL_RCC_LSI2_Enable();
```

Comment:
LSI clock source is a valid clock source for WB10xx & WB15xx.
Do we just not support this feature for these MCUs?